manticore/core/manticore.py

```diff
@@ -39,16 +43,19 @@ class ManticoreBase(Eventful):
 
     _published_events = {'start_run', 'finish_run', 'generate_testcase'}
 
-    def __init__(self, initial_state, workspace_url=None, policy='random', **kwargs):
+    def __init__(self, initial_state, workspace_url=None, policy='random', timeout=None, **kwargs):
```

Comment:
Do we need the timeout here if it is already a global config value ?

Reply:
It's the question where should the user specify it if they want to do so. Do you think they should use `config.core.timeout` instead?

Reply:
Let's leave it as an argument here as someone might want to use it when playing with scripts.

Reply:
Ok. Note though that she may still need to do some config.py setting even from a script.